src/Pact/Repl/Lib.hs

```diff
@@ -144,6 +145,11 @@ replDefs = ("Repl",
      "Output VALUE to terminal as unquoted, unescaped text."
      ,defZRNative "env-hash" envHash (funType tTyString [("hash",tTyString)])
      "Set current transaction hash. HASH must be a valid BLAKE2b 512-bit hash. `(env-hash (hash \"hello\"))`"
+     ,defZNative "test-capability" testCapability
+      (funType tTyString [("capability", TyFun $ funType' tTyBool [])]) $
+     "Specify and request grant of CAPABILITY. Once granted, CAPABILITY and any composed capabilities are in scope " <>
+     "for the rest of the transaction. Allows direct invocation of capabilities, which is not available in the " <>
+     "blockchain environment. `$(grant-capability (TRANSFER) (my-module.transfer sender receiver 1.0))`"
```

Comment:
Example is not correct, has body form in it.

Reply:
Ack thanks